doc/user/content/sql/system-catalog.md

```diff
@@ -342,6 +342,10 @@ Field          | Type       | Meaning
 The `mz_objects` view contains a row for each table, source, view, materialized view, sink,
 index, connection, secret, type, and function in the system.
 
+IDs for all objects represented in `mz_objects` share a namespace -
```

Comment:
Light suggestion to remove both the hyphen (nee em dash) and e.g.:

> IDs for all objects represented in `mz_objects` share a namespace. If there is a view with ID u1 ...

If you want to keep them, let's at least use a real em dash.